bom-weekly/pom.xml

```diff
@@ -462,6 +462,12 @@
         <artifactId>credentials-binding</artifactId>
         <version>631.v861c06d062b_4</version>
       </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>cvs</artifactId>
+        <!-- TODO https://github.com/jenkinsci/cvs-plugin/pull/87 -->
+        <version>2.20-rc444.7b_3b_2d526eee</version>
```

Comment:
Currently blocked on merge/release of https://github.com/jenkinsci/cvs-plugin/pull/87